utils/users.js

```diff
@@ -121,24 +121,14 @@ function getPaginationLink(query, cursor, documentId) {
  * @param allUsers {Array} - list of total users from firebase
  *
  */
-function getFilteredUsers(allPRs, allUsers) {
+function getFilteredUsernames(allPRs) {
   const uniqueUsersInOrder = new Set();
 
   allPRs?.forEach((element) => {
     uniqueUsersInOrder.add(element.username);
   });
 
-  const usersWithDetails = [];
-
-  uniqueUsersInOrder.forEach((username) => {
-    const userDetails = allUsers.find((user) => user.github_id === username);
-
-    if (userDetails) {
-      usersWithDetails.push(userDetails);
-    }
-  });
-
-  return usersWithDetails;
+  return Array.from(uniqueUsersInOrder);
```

Comment:
`uniqueUsersInOrder` is misleading variable name, as it's a Set and no order is guaranteed in it. We should rename it.